exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -97,97 +96,75 @@
   public static final short MAX_VALIDATOR_COUNT_WITH_ENABLED_TIME_SERVICE = 3;
   @VisibleForTesting
   static final short MAX_SERVICE_NUMBER = 256;
+  @VisibleForTesting
+  static final int MAX_SERVICE_INSTANCE_ID = 1023;
```

Comment:
Does the core have such restriction that we have to enforce?

Reply:
TestKit services are created as builtin services and there is a max id for those - https://github.com/exonum/exonum/blob/dynamic_services/exonum/src/runtime/dispatcher/mod.rs#L54